chef-automate-ha/azuredeploy.json

```diff
@@ -252,7 +252,8 @@
         "felbPublicIPAddressID": "[resourceId('Microsoft.Network/publicIPAddresses',variables('networkSettings').felbPublicIPAddressName)]",
         "chefAutoPublicIPAddressID": "[resourceId('Microsoft.Network/publicIPAddresses',variables('networkSettings').chefAutoPublicIPAddressName)]",
         "deploymentApiVersion": "2015-01-01",
-        "prefix": "[uniqueString(resourceGroup().id)]"
+        "prefix": "[uniqueString(resourceGroup().id)]",
+        "provider": "[toUpper('33194f91-eb5f-4110-827a-e95f640a9e46')]"
```

Comment:
Yes, this is a tracking device that our DX Liaison @sebastus has asked us to put in all our templates.